### DISCUSSION TITLE
Detect "Fall"

### DISCUSSION THREAD

**Discussion by KyllianGamer** [Minestom/Minestom#1841] [Help] [answered]
Any event / way that tells when the player is grounded after it wasn't (the previous tick)?

**Comment by TogAr2** [comment, marked as answer]
Use `PlayerMoveEvent`. You can check if the player has moved onto the ground by using `event.isOnGround()`, and whether the player was already on the ground by using `event.getPlayer().isOnGround()`

**Reply by VendoAU** [reply]
Just be aware that this is sent from the player and can be spoofed using a modified client.
